README.md

```diff
@@ -38,7 +38,7 @@ If you want to test recent bugfixes or features that have not been packaged in a
     }
 
     dependencies {
-      compile 'io.realm:realm-android:0.83.0-SNAPSHOT'
+      compile 'io.realm:realm-android:0.82.1-SNAPSHOT'
```

Comment:
Yes, it should. I thought i could just change it to 0.82.2 directly when do the release since this won't be merged to the master, no snapshot build will be triggered